### ISSUE TITLE
Precedence problem in CcpdflConverter

### ISSUE THREAD

**Issue by simonspa** [eudaq/eudaq#182]
Hi @theresaobermann 

there is something you probably want to have a look at:
```
/ilcsoft/eudaq/main/lib/plugins/CcpdlfConverterPlugin.cc:61:25: warning: & has lower precedence than !=; != will be
      evaluated first [-Wparentheses]
            if (data[3] & 0x80 !=0x80) return (unsigned) -1;
                        ^~~~~~~~~~~~~
/ilcsoft/eudaq/main/lib/plugins/CcpdlfConverterPlugin.cc:61:25: note: place parentheses around the '!=' expression to
      silence this warning
            if (data[3] & 0x80 !=0x80) return (unsigned) -1;
                        ^
                          (          )
/ilcsoft/eudaq/main/lib/plugins/CcpdlfConverterPlugin.cc:61:25: note: place parentheses around the & expression to evaluate
      it first
            if (data[3] & 0x80 !=0x80) return (unsigned) -1;
                        ^
                (             )

```

I'm pretty sure you want to apply the bit mask to the data word...

**Comment by arummler** [comment]
Reminder due to MSVC warnings cleanup.

**Comment by arummler** [comment]
@theresaobermann 
C:\projects\eudaq\main\lib\plugins\CcpdlfConverterPlugin.cc(61): warning C4554: '&' : check operator precedence for possible error; use parentheses to clarify precedence [C:\projects\eudaq\build\main\lib\EUDAQ.vcxproj]

The other warnings are gone.

**Comment by arummler** [comment]
Fixed in d84c09fe47378da8ab000c6f22aa2c6c217b81eb
@theresaobermann Please check
